### PR TITLE
P4 885 part 2 data fixup

### PR DESCRIPTION
**Pull request by starswan**

### CODE REVIEW

**Comment by martyn-w** [comment]
@starswan could you pull in the latest master into your branch ?

File: app/serializers/profile_serializer.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  )
+
+  # although we are serialising a 'profile', to the outside world this is a person.
+  type 'people'
+
+  has_one :ethnicity, serializer: EthnicitySerializer
+  has_one :gender, serializer: GenderSerializer
+
+  def id
+    object.person.id
```
Comment (martyn-w) [inline app/serializers/profile_serializer.rb]: @starswan should this definitely be the `person.id` and not the `profile.id` ? There is a danger that different profiles for the same person could be returned - each with the same apparent id. That might confuse api consumers?

Comment (starswan) [reply app/serializers/profile_serializer.rb]: Yes - there's no profile endpoint. The profile returned by the person endpoint is always the 'latest_profile' (yes I'm not sure whether this works either)

**Comment by martyn-w** [comment]
@starswan can you pull in the latest master branch?

**Pull request merged**